### ISSUE TITLE
胡莺夕 Plan for class

### ISSUE THREAD

**Issue by huyingxi** [BUPT-CAD/wora-app#5]
**Name** : huyingxi (胡莺夕)
**School**: BUPT(北邮)
**Platform** : IOS
**Data** : Undetermined(待定)
**Deadline**:7月1号

**Comment by huan** [comment]
No, As I told you in the class: the deadline must be a specific date. 

Please update.

**Comment by huyingxi** [comment]
my wora github link : 
       https://github.com/huyingxi/wora-app.git
Thx!

**Comment by huan** [comment]
I saw you have updated the deadline.

However, besides the GitHub link, you should also provide a video in GIF format that capture your screen and demos your app.

Please update, by posting your screen capture video in GIF file in this issue.

Thanks.

**Comment by huyingxi** [comment]
I have uploaded a mov file in this path : https://github.com/huyingxi/wora-app/wora_hyx.mov
Because the GIF format file that you specified is too large( over 25mb ), so I uploaded another MOV format file.

**Comment by huan** [comment]
The GIF format file I specified can be the only 3MB for your video, using the following parameters:
1. Resolution with height: 500px
1. FPS: 5 per second

I made this from your video(3MB):

![wora-hyx](https://user-images.githubusercontent.com/1361891/27260946-3ec3c41c-53fe-11e7-8d8a-9bbc34ae4b32.gif)

Please tell the other classmates for me if they also have trouble with how to make the GIF video as I required. Thanks!
